### ISSUE TITLE
I think there are some bad examples in the portuguese version

### ISSUE THREAD

**Issue by yudi-azvd** [conventional-commits/conventionalcommits.org#457]
In the [portuguese version of conventional commits](https://www.conventionalcommits.org/pt-br/v1.0.0/), I think there are some bad examples like the following ([link](https://www.conventionalcommits.org/pt-br/v1.0.0/#mensagem-de-commit-com--para-chamar-a-aten%C3%A7%C3%A3o-para-quebra-a-compatibilidade)):

```
## Mensagem de commit com ! para chamar a atenção para quebra a compatibilidade

refactor!: remove suporte para Node 6
```

Which translates to

```
## Commit message with ! to draw attention to breaking change

refactor!: removes support for Node 6
```

Maybe this example is not adequate since refactoring means changing the internal structure _without modifying observable behavior_ (Refactoring, Kent Beck). Refactor and breaking changes shouldn't be in the same commit/PR, right?

I'm no expert in software development or refactoring, I'm still learning things. So I'm if I understood something, please tell me :D

**Comment by yudi-azvd** [comment]
I've just read the [summary](https://www.conventionalcommits.org/en/v1.0.0/#summary) and it says A BREAKING CHANGE can be part of commits of any _type_.

Is this correct even for `refactor:` type? Can someone make it clearer?

Also, the reasoning presented in my previous comment was accepted in this [PR](https://github.com/conventional-commits/conventionalcommits.org/pull/350).

**Comment by javier-godoy** [comment]
See #229

**Comment by yudi-azvd** [comment]
Ok... i've read it. Should I open a PR correcting it?

**Comment by damianopetrungaro** [comment]
Sure @yudi-azvd ! Feel free to open a PR suggesting a better example! Thanks for noticing it as well :D

**Comment by yudi-azvd** [comment]
Ok :D One more thing. When correcting, should I modify `v1.0.0/index.pt-br.md` or create a new one in, say, `v1.0.1/...`?

**Comment by yudi-azvd** [comment]
Closed by #458.
